src/app/manage-learn/core/services/api.service.ts

```diff
@@ -47,6 +47,8 @@ export class ApiService {
           headers: new HttpHeaders({
             'x-auth-token': session ? session.access_token : "",
             'x-authenticated-user-token': session ? session.access_token : "",
+            'appName':'diksha',
+            'appVersion':''
```

Comment:
@kiranharidas187 is it mandatory to send the appName and appVersion. if Yes then instead of sending appName send the appId and version like following
 'X-App-Id': <producer_id> 
'X-App-Ver': <app_version>